personalsite/src/App.js

```diff
@@ -0,0 +1,54 @@
+import logo from './logo.svg';
+import './App.css';
+import React from 'react';
+
```

Comment:
You should import in navbar to include it in the app! 
`import Navbar from './Components/Navbar/Navbar';`